lib/Refactoring/LocalRename.cpp

```diff
@@ -283,21 +299,30 @@ swift::refactoring::getRenameInfo(ResolvedCursorInfoPtr cursorInfo) {
 }
 
 class RenameRangeCollector : public IndexDataConsumer {
+  StringRef USR;
+  StringRef newName;
+  std::unique_ptr<StringScratchSpace> stringStorage;
+  std::vector<RenameLoc> locations;
```

Comment:
Is the collector actually used again with `newName`, or do we always end up doing the syntactic rename after we have locations?